docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -600,6 +600,13 @@ Query vector. Must have the same number of dimensions as the vector field you
 are searching against.
 end::knn-query-vector[]
 
+tag::knn-query-vector-builder[]
+A configuration object indicating how to build a query_vector before executing
+the request. You must provide a `query_vector_builder` or `query_vector`, but
+not both. Refer to <<knn-semantic-search>> to learn more.
+end::knn-query-vector-builder[]
+
+
```

Comment:
I think you need to update `docs/reference/query-dsl/knn-query.asciidoc` as well to include `knn-query-vector-builder`

Reply:
can I use `include` directive inside `docs/reference/query-dsl/knn-query.asciidoc` ?

Reply:
You should be able to use `include`